IMLS/imls_scraper.py

```diff
@@ -84,7 +88,7 @@ def latest_date(self) -> datetime:
     data_page_url='https://www.imls.gov/research-evaluation/data-collection/public-libraries-survey',
     data_format="DF",
     extract_usecols=[
-       'LIBID', 'LIBNAME', 'ADDRESS', 'CITY', 'STABR', 'ZIP', 'PHONE'
+       'LIBID', 'LIBNAME', 'ADDRESS', 'CITY', 'STABR', 'ZIP', 'PHONE', 'STATSTRU'
```

Comment:
I feel the this column will appear in final tmpILMS table , we need to drop it after the notes column is populated.

Reply:
done

Reply:
Thanks